models/dashboard.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 // Dashboard dashboard
+// Example: isResource
```

Comment:
Please add only those files related to the escalation chain.

Reply:
Done